.travis.yml

```diff
@@ -34,8 +34,8 @@ script:
   - export GOOS=$TRAVIS_GOOS
   - export CGO_ENABLED=$TRAVIS_CGO_ENABLED
   - DCO_VERBOSITY=-q script/validate/dco
-  - GOOS=linux script/setup/install-dev-tools
-  - script/validate/vendor
+  - GOOS=linux GO111MODULE=off script/setup/install-dev-tools
```

Comment:
@SuperQ FYI. This solves the problem you were facing.

Reply:
Great! Thanks! I guess I can close my PR then.